### PR TITLE
{chem}[foss/2023a] DL_POLY_4 v5.1.0

### PR DESCRIPTION
**Pull request by WilleBell**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by WilleBell** [comment]
Test report by @WilleBell
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node601.hydra.os - Linux Rocky Linux 8.9, x86_64, AMD EPYC 9384X 32-Core Processor, Python 3.6.8
See https://gist.github.com/WilleBell/d8fe4e7f8bdaede2f3f9b9e161c9a8b0 for a full test report.

File: easybuild/easyconfigs/d/DL_POLY_4/DL_POLY_4-5.1.0-foss-2023a.eb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+source_urls = ['https://gitlab.com/ccp5/dl-poly/-/archive/%(version)s/']
+sources = ['dl_poly_%(version)s.tar.bz2']
+checksums = ['da5364986cd71e047e080753f6ca75135bf19bd5607770b839dea3734c2fdfaa']
+
+builddependencies = [('CMake', '3.26.3')]
+
+sanity_check_paths = {
+    'files': ['bin/DLPOLY.Z'],
+    'dirs': []
+}
```
Comment (smoors) [inline easybuild/easyconfigs/d/DL_POLY_4/DL_POLY_4-5.1.0-foss-2023a.eb:20]: can you add a `sanity_check_commands` here? for example
```
sanity_check_commands = ['DLPOLY.Z -h']
```

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen3

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @WilleBell!

**Pull request merged**